cmd/cartesi-rollups-node/main.go

```diff
@@ -43,6 +45,9 @@ func main() {
 	slog.SetDefault(logger)
 	slog.Info("Starting the Cartesi Rollups Node", "version", buildVersion, "config", config)
 
+	// run database migrations
+	repository.RunMigrations(fmt.Sprintf("%v?sslmode=disable", config.PostgresEndpoint.Value))
+
```

Comment:
We should add a `repositoy.SetupDatabaseConfig` call here. So every time the node starts it tries to setup the database.